githooks.sh

```diff
@@ -2,7 +2,9 @@
 
 if [ ! -f .git/hooks/pre-commit ]
 then
-  ln lint.sh .git/hooks/pre-commit
+  pushd .git/hooks
+  ln -s ../../lint.sh .git/hooks/pre-commit
+  popd
```

Comment:
This was required to get lint to pass when switching branches. Previously lint.sh was hard linked to pre-commit so the hook did not reflect the current branch's lint.sh.